README.md

```diff
@@ -1095,15 +1095,15 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 FROM node:14.4.0 AS build
 
 COPY . .
-RUN npm install && npm run build
+RUN npm ci && npm run build
```

Comment:
@BrunoScheufler You wrote this so just cc'ing you, I guess 'ci' is better here and more consistent with the rest of our recommendations, right?

Reply:
Sure, we can change it to npm ci 👍